dvc/command/experiments.py

```diff
@@ -308,44 +321,57 @@ def _experiments_table(all_experiments, **kwargs):
         include_params=include_params,
         exclude_params=exclude_params,
     )
+    metric_headers = _normalize_headers(metric_names)
+    param_headers = _normalize_headers(param_names)
```

Comment:
It could have simplified a lot of things here if we were working with a flattened/normalized params and metrics data.
Working with flattened headers but with expanded/normal params and metrics might cause unintended bugs. 
Leaving it for now though.

Reply:
Using the flattened data might make this code cleaner thanks to the Table, but as-is, it can provide very little advantage.

Reply:
And, since experiments are more like nested tables than a simple table, the abstraction does not work well for the experiments. For example, sorting the best experiment.